ci/generate-docs.sh

```diff
@@ -30,23 +31,30 @@ CLONE_URL="git@github.com:spatialos/gdk-for-unity.git"
 CODE_DIR="${TMP_DIR}/code"
 git clone "${CLONE_URL}" "${CODE_DIR}" --branch "${TAG}" --depth 1
 
-# Clone docgen repo.
-DOCGEN_CLONE_URL="git@github.com:improbable/gdk-for-unity-docgen.git"
-DOCGEN_DIR="${TMP_DIR}/docgen"
-git clone "${DOCGEN_CLONE_URL}" "${DOCGEN_DIR}"
-
-pushd "${DOCGEN_DIR}"
-    docker build . --tag docgen
-popd
-
 docker run --rm \
     -v "${CODE_DIR}/workers/unity/Packages:/input" \
     -v "${OUTPUT_DIR}:/output" \
-    docgen \
+    "eu.gcr.io/io-crafty-shelter/gdk-for-unity-docgen:${DOCGEN_PIN}" \
         --target-namespace="Improbable.Gdk" \
         --namespace-filter=".*EditmodeTests" \
         --namespace-filter=".*DeploymentLauncher" \
         --namespace-filter=".*PlaymodeTests" \
         --git-tag="${TAG}"
 
-# TODO: Trigger Filip's pipeline when that's live
+pushd "${OUTPUT_DIR}"
+    buildkite-agent artifact upload "**/*.md"
+popd
+
+function generate_step() {
```

Comment:
very small nit: I personally prefer to have the functions grouped together at the start

Reply:
Sure, can move it